shell/platform/windows/win32_window.cc

```diff
@@ -0,0 +1,220 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/win32_window.h"
+
+namespace flutter {
+
+Win32Window::Win32Window() {
+  dpi_helper_ = std::make_unique<Win32DpiHelper>();
```

Comment:
This can go in the initializer list rather than the body, can't it? (Or even at the declaration point, which would be more consistent?)

Reply:
Fixed in next commit.